Week2/Pascal's_triangle_4_fixes/src/Triangle.java

```diff
@@ -16,32 +17,53 @@ private static Long getFactorial(Integer k) {
 		return factorial;
 	}
 	
-	private static Long getMuntiplication(Integer a, Integer b) {
+	private static Long getMultiplicationAtoB(Integer a, Integer b) {
 		Long result = 1L;
 		for (int i = a; i <= b; i++) {
 			result = i * result;
 		}
 		return result;
 	}
 	
-	private static Long C(int n, int k) {
-		Integer max = Integer.max(k, n-k);
-		Integer min = Integer.min(k, n-k);
-		return getMuntiplication(max + 1, n) / (getFactorial(min));
-	}
-	
-	public void printTriangle(int n) {
+		
+	public void fix1_SoutToSBuilder(int _n) {
```

Comment:
please don't name it like this anymore. Consider rather creating separating classes/packages.
_n naming is not appropriate - this is not a Python.